packages/app/src/runner/event-manager.ts

```diff
@@ -131,11 +130,6 @@ export class EventManager {
     })
 
     this.ws.on('specs:changed', ({ specs, testingType }) => {
-      // do not emit the event if e2e runner is not displaying an inline spec list.
-      if (testingType === 'e2e' && state.useInlineSpecList === false) {
-        return
-      }
-
```

Comment:
should this have been maintained? I'm not seeing a test around this, is there an existing one?

Reply:
Good question... here's the historical context. Before it was decided to go all in on the Vite/Vue rewrite, I wanted to just rework the existing app without touching the stack (React, Webpack, event emitter instead of GraphQL). I actually added an inline spec list to E2E as part of that, which was hidden behind a feature flag that never saw the light of day. The idea was `runner-shared` would have 99% of the code, the `runner` and `runner-ct` would have been the 1% of code differing between the two.

You can see where that was added and a sweet demo here: https://github.com/cypress-io/cypress/pull/17069. There is a test that isn't running, since we are not running the `runner` tests on CI (since we are not using that code anymore). For interest, the test is here: https://github.com/cypress-io/cypress/blob/develop/packages/runner/cypress/integration/runner.ui.spec.js#L409.

I'm still wondering if reworking the existing infrastructure with React/Webpack would have been the better move. :man_shrugging:.

To answer your question: this code is now obsolete, since we will always be showing an inline spec list in the runner, so that's why I deleted it.

Reply:
huh! thanks for all the info! Sounds like this was the first step toward unification's shared spec list! I would LOVE if we could run e2e & component tests within the same frame. Maybe some day 😄

Reply:
Not sure on the technical complexity/value but it sure would be nice. Obvious problem is we'd need to start the dev server regardless, since we wouldn't know if you are using E2E or CT. Another battle for another day.